packages/web-components/fast-foundation/src/badge/badge.template.ts

```diff
@@ -7,14 +7,7 @@ import { Badge } from "./badge";
  */
 export const BadgeTemplate = html<Badge>`
     <template class="${x => (x.circular ? "circular" : "")}">
-        <div
-            class="control"
-            part="control"
-            style="${x =>
-                x.fill || x.color
-                    ? `background-color: var(--badge-fill-${x.fill}); color: var(--badge-color-${x.color})`
-                    : void 0}"
-        >
+        <div class="control" part="control" style="${x => x.generateBadgeStyle()}">
```

Comment:
Does this update if fill or color change?

Reply:
I tested by changing color in the DOM and it updated with the expected colors.